src/Context/JsonContext.php

```diff
@@ -332,8 +332,6 @@ public function theJsonShouldBeValidAccordingToTheSchema($filename)
      */
     public function theJsonShouldBeInvalidAccordingToTheSchema($filename)
     {
-        $this->checkSchemaFile($filename);
```

Comment:
This line is necessary, because if it’s failed in `theJsonShouldBeValidAccordingToTheSchema` method, the `not` function makes it’s ok.